modules/Search/components/Facet.tsx

```diff
@@ -71,7 +74,12 @@ function Facet({ attribute, items, refine }: RefinementListProvided & Refinement
                     <Disclosure.Panel className={styles.panel}>
                         <ul className={styles.list}>
                             {visibleItems.map((item) => (
-                                <li key={`${attribute}_${item.label}`} className={styles.listItem}>
+                                // eslint-disable-next-line
```

Comment:
✖️ 
Why is eslint disabled completely here? 🤔

Reply:
Updated!